frontend/packages/pipelines-plugin/src/components/import/pipeline/PipelineSection.scss

```diff
@@ -0,0 +1,3 @@
+.odc-form-section-pipeline {
```

Comment:
now that we have pipelines as a separate package, I think it will be better if we should use `pipeline-` prefix instead of `odc`, thoughts?

Reply:
Agreed, but let's create a separate task for this as there's many instances in the package that need updating.
Maybe something short like `op` (OpenShift Pipelines) or `tp` (Tekton Pipelines).

Reply:
Created an issue for this https://issues.redhat.com/browse/ODC-5253